auth/tests/unit/cognito-client-oidc.unit.test.ts

```diff
@@ -86,7 +86,7 @@ describe("Set up the Cognito User Pool OIDC client", () => {
     template.hasResourceProperties("AWS::Cognito::UserPoolClient", {
       AccessTokenValidity: 300, //5 minutes
       IdTokenValidity: 3600, // one hr
-      RefreshTokenValidity: 2592000, // 30 days
+      RefreshTokenValidity: 86400, // 24 hrs
```

Comment:
There's an acc test that needs updating see https://github.com/alphagov/govuk-mobile-backend/pull/166/files#diff-157c4e23808dcd4b87aa67d4626cea6b2a09a5a0ac3595f288d77332bdea33f4